packages/widget/src/types/events.ts

```diff
@@ -6,6 +6,7 @@ export enum WidgetEvent {
   RouteExecutionCompleted = 'routeExecutionCompleted',
   RouteExecutionFailed = 'routeExecutionFailed',
   RouteHighValueLoss = 'routeHighValueLoss',
+  SafeRouteInitiated = 'safeRouteInitiated',
```

Comment:
If we want a special event when multisig transaction started, we should name it as `MultisigTransactionStarted`. As we discussed, we must avoid using wallet specific terminology here.